apps/omg/lib/omg/utxo/exit.ex

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Utxo.Exit do
+  @moduledoc """
+  Represents a (U)TXO exit.
+  """
+
+  defstruct [:rootchain_txhash, :utxo_pos]
```

Comment:
eh, sorry for missing this earlier: `root_chain_txhash`

Reply:
I deleted this file already. But throughout have been using `rootchain_txhash` and `childchain_utxohash` both in the DB and code.